integration/networking/port_mapping_linux_test.go

```diff
@@ -408,6 +409,92 @@ func TestAccessPublishedPortFromRemoteHost(t *testing.T) {
 	}
 }
 
+func TestAccessPublishedPortFromCtr(t *testing.T) {
+	skip.If(t, testEnv.IsRootless)
```

Comment:
nit; we should add comments explaining why a skip is in place (code comment and); third argument of "skip.If" can be used to add a custom message for the skip